Services/Infrastructure/OJS.Services.Infrastructure/HttpClients/Implementations/SulsPlatformHttpClientService.cs

```diff
@@ -54,7 +54,7 @@ public Task<ExternalDataRetrievalResult<TData>> GetAsync<TData>(object requestDa
             var externalDataResult = new ExternalDataRetrievalResult<TData>();
 
             var queryStringSeparator = GetQueryStringSeparator(endpoint);
-            var requestUrl = $"/{endpoint}{queryStringSeparator}apiKey={this.apiKey}";
+            var requestUrl = $"{endpoint}{queryStringSeparator}apiKey={this.apiKey}";
```

Comment:
IMO, it's always best to start subUrls with a leading slash, i.e. `/....`

Reply:
This results in an error: 
```
System.Net.Sockets.SocketException (11001): No such host is known. (submissions:80)
   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken) 
```
   
   because the request url = ```//submissions/add?apiKey=1234```
   
   either this slash or all slashes is the begginings of urls in GlobalConstants.Urls must be removed:
   
![image](https://user-images.githubusercontent.com/19411274/161572460-8692cf84-c155-46f3-bdcf-6b79cae262df.png)